tracer/test/Datadog.Trace.TestHelpers/TracerRestorerAttribute.cs

```diff
@@ -19,7 +19,7 @@ public class TracerRestorerAttribute : BeforeAfterTestAttribute
 
         public override void Before(MethodInfo methodUnderTest)
         {
-            _tracer = Tracer.Instance;
+            _tracer = Tracer.InternalInstance;
```

Comment:
Technically we don't _need_ to change the usages inside the tests I think - we don't enforce `[PublicApi]`, but happy that it probably makes sense to!